### DISCUSSION TITLE
How to adjust space between date and new description in the news collection?

### DISCUSSION THREAD

**Discussion by hy-zhao23** [alshedivat/al-folio#1883] [Q&A] [answered]
<img width="269" alt="image" src="https://github.com/alshedivat/al-folio/assets/52598758/1e4d25cf-ed5c-4be7-a5b4-56fdfdc5f01a">

After I increase the max_length of the webpage, the space between the date and new description is very large. Is there a way to decrease it? Also, the vertical space between news are kind of large too. Seems there are the paragraph space. How can I make it into line space?

**Comment by matonoli** [comment, marked as answer]
In file _includes/news.html you can replace the line with the `<th>` tag with something like
`<th scope="row" style="padding: 0.2rem; width: 17%">{{ item.date | date: "%b %-d, %Y" }}</th>`
